coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,20 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
 from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
 
 
 def main(log_printer=None, section: Section=None):
+    if log_printer is not None:
+        logging.warning('Using log_printer is deprecated, '
+                        'use python inbuilt logging instead.')
     configure_logging()
```

Comment:
oh hasn't `configure_logging` to be run before warn the user of `log_printer` deprecation?